css/index.css

```diff
@@ -1,66 +1,213 @@
 
+
+
+  /* Base styles */
 body {
-    background-color: rgb(142, 142, 158);
-    color: white;
-    justify-content: center;
-    align-items: center;
-    height: 100vh;
     margin: 0;
-    padding-left: 50px;
-    padding-top: 100px;
-    padding-bottom: 50px;
-    padding-right: 60px;
-    /* text-align: center; */
-    /* display: flex; */
-  }
-  .wrapper { 
-    margin: auto;
-    width: 1000px;
+    font-family: Roboto, sans-serif;
+    font-size: 16px; /* Adjust base font size */
+    line-height: 1.5; /* Improve readability */
   }
-.img-coding { 
-width: 400px;
-height: 200px;
+
+  /* Desktop first layout*/
+html { 
+    scroll-behavior: smooth;
+}
+header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 999;
 }
 
-h1 { 
+.header {
+    padding: 40px;
+    text-align: center;
+    background: gold;
+    margin: 30px;
     font-size: 20px;
+  }
+
+#skills ul {
+    padding-left: 10px;
+    background-color: lightcoral;
+  }
+
+   #About h2 {
+    font-size: 60px; 
+    font-style: italic; 
     color: green;
+  }
+  .image-container {
+    width: 650px; 
+    height: 250px; 
+    border-radius: 50%;
+    overflow: hidden;
+    margin: 0 auto; 
+}
+
+.profile-image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
 }
 
+
+body { 
+ /*background-color: rgb(142, 142, 158);*/
+ background: linear-gradient(to top, #a8c9e2, #f0f9ff);
+ color: white;
+ justify-content: center;
+ align-items: center;
+ height: 100vh;
+ margin: 0;
+ padding-left: 50px;
+ padding-top: 100px;
+ padding-bottom: 50px;
+ padding-right: 60px;
+ font-family: Roboto;
+
+}
+ h1 { 
+    font-size: 20px;
+    color: green;
+    font-weight: bold;
+ }
+
 nav ul li { 
-    display: inline;
+    display:inline-flex;
     color: lightblue;
+    color: rgb(141, 145, 141);
     font-size: 25px;
-    margin-inline: 30px;
+    margin-inline: 90px;
+    position: sticky;
+    top: 0;
+    font-weight: bold;
+    background-color: #fff; 
+    box-shadow:navajowhite rgba(110, 230, 13, 0.1);
+    background-color: gold; 
+    box-shadow:goldenrod rgba(110, 230, 1);
+    border-radius: 30%;
+    text-decoration: none;
 }
-
-#about, #experience, #connect { 
-    margin: 20px;
-    padding: 20px;
+nav {
+    color: green;
 }
```

Comment:
As far as I can tell, the nav bar doesn't have a color. Is it rendering correctly for you?